### PR TITLE
fix(logs.detail.inputs.add): remove harcoded TCP port

### PR DESCRIPTION
**Pull request by Heiiji**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-30658
| License          | BSD 3-Clause

## Description

Forward TCP selection on next step of data-gathering adding because required on warning wording.

### :flags: Translations

- [x] TRANS-35188

### CODE REVIEW

File: packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default class LogsInputsAddConfigureCtrl {
     this.$translate = $translate;
     this.serviceName = this.$stateParams.serviceName;
     this.inputId = this.$stateParams.inputId;
+    this.exposedPort = this.$stateParams.exposedPort;
```
Comment (radireddy) [inline packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js:22]:
```suggestion
    this.exposedPort = this.$transition$.params().exposedPort;
```

Comment (cbourgois) [reply packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js:22]: As `$stateParams` is used before I think we could keep it.

**Review by cbourgois — APPROVED** [review]

**Review by radireddy — DISMISSED** [review]

**Comment by ovh-cds** [comment]
CDS Report translator-push#16981.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#16982.0 ✘  
* 
  * push ✘

**Review by antleblanc — APPROVED** [review]

**Review by marie-j — DISMISSED** [review]

**Comment by marie-j** [comment]
Hello @Heiiji, 

Could you please amend your commit to specify the ticket reference ? 

Thanks

**Review by marie-j — APPROVED** [review]

**Pull request merged**